e2e_test/source/basic/nosim_kafka.slt

```diff
@@ -1,3 +1,6 @@
+control substitution on
```

Comment:
Only try `RISEDEV_SCHEMA_REGISTRY_URL` in this test. Not bother to modify others.